test/cmdlineTests.sh

```diff
@@ -173,8 +181,25 @@ TMPDIR=$(mktemp -d)
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/ docs
     for f in *.sol
     do
+        # The contributors guide uses syntax tests, but we cannot
```

Comment:
Ah, here we could use ``isoltest``, once it accepts files/directories as arguments.